src/vm/scanner.h

```diff
@@ -35,7 +35,7 @@ typedef enum {
     TOKEN_CLASS, TOKEN_ABSTRACT, TOKEN_TRAIT, TOKEN_USE, TOKEN_STATIC,
     TOKEN_PRIVATE, TOKEN_THIS, TOKEN_SUPER, TOKEN_DEF, TOKEN_AS,
     TOKEN_ENUM,
-    TOKEN_IF, TOKEN_AND, TOKEN_ELSE, TOKEN_OR,
+    TOKEN_IF, TOKEN_AND, TOKEN_ELSE, TOKEN_OR, TOKEN_MATCH,TOKEN_CASE,
```

Comment:
Small formatting change

```suggestion
    TOKEN_IF, TOKEN_AND, TOKEN_ELSE, TOKEN_OR, TOKEN_MATCH ,TOKEN_CASE,
```